pkg/polaris/ctl_polaris.go

```diff
@@ -77,6 +77,33 @@ type FlagTree struct {
 	coverityLicensePath string
 }
 
+// DefaultFlagTree ...
+// [Dev Note]: These should match the Helm Chart's Values.yaml
+var DefaultFlagTree = FlagTree{
+	// Version
+	Version: "2020.03",
+	// domain-name specific flags
+	IngressClass: "nginx",
+	// license related flags
+	// smtp related flags
+	SMTPPort:                 2525,
+	SMTPTlsMode:              "require-starttls",
+	SMTPTlsTrustedHosts:      "*",
+	SMTPTlsIgnoreInvalidCert: false,
+	// postgres specific flags
+	PostgresInternal: false,
+	PostgresPort:     5432,
+	// size parameters
+	EventstoreSize:     EVENTSTORE_PV_SIZE,
+	MongoDBSize:        MONGODB_PV_SIZE,
+	DownloadServerSize: DOWNLOAD_SERVER_PV_SIZE,
+	UploadServerSize:   UPLOAD_SERVER_PV_SIZE,
+	PostgresSize:       POSTGRES_PV_SIZE,
+	// reporting related flags
+	EnableReporting:   false,
+	ReportStorageSize: REPORT_STORAGE_PV_SIZE,
+}
```

Comment:
Enable persistent storage is missing?

Reply:
There isn't a flag for enable persistent storage